.github/workflows/main.yml

```diff
@@ -121,7 +141,7 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           # Upload the dist folder. Give it a name according to the OS it was built for.
-          name: dist-ubuntu-xenial
+          name: dist-ubuntu-bionic
```

Comment:
Is this needed?  Will it effect the portability of the resulting binaries (e.g bump the min glibc version?)

Reply:
The `Dockerfile` has been updated to `bionic` a long time ago, this is just changing the outdated name to reflect the status quo :)